notification/test/unit/lambda.spec.js

```diff
@@ -0,0 +1,57 @@
+const sinon = require('sinon')
+const chai = require('chai')
+const { handler } = require('../../src/lambda')
+const notificationHandler = require('../../src/handler/notification/notification.handler')
+const setup = require('../../src/config/init/ensure-interface-interaction-custom-type')
+const logger = require('../../src/utils/logger')
+
+const { expect, assert } = chai
+
+chai.use(require('chai-as-promised'))
+
+describe('Lambda handler', () => {
+  let ensureResourcesStub
+
+  beforeEach(() => {
+    ensureResourcesStub = sinon.stub(setup, 'ensureInterfaceInteractionCustomType').returns(undefined)
+  })
```

Comment:
Could you add a line break between this `Each`s?

Reply:
Done